src/main/java/net/minecraftforge/event/ForgeEventFactory.java

```diff
@@ -306,6 +307,11 @@ public static int getMaxSpawnPackSize(EntityLiving entity)
         return maxCanSpawnEvent.getResult() == Result.ALLOW ? maxCanSpawnEvent.getMaxPackSize() : entity.getMaxSpawnedInChunk();
     }
 
+    public static void onFoundChunksForSpawning(FoundChunksForSpawningEvent.Builder builder)
+    {
+        MinecraftForge.EVENT_BUS.post(new FoundChunksForSpawningEvent(builder));
```

Comment:
Since java is oop, i'd suggest you moving this functionality to an instance method of the builder object.